server/game/cards/13-O4B/DocHolliday.js

```diff
@@ -0,0 +1,57 @@
+const LegendCard = require('../../legendcard.js');
+
+class DocHolliday extends LegendCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.game.shootout && this.game.shootout.leaderPlayer === this.owner && !this.isSpellcasterInLeaderPosse(),
+            match: this.owner,
+            effect: [
+                ability.effects.modifyPosseStudBonus(-2)
+            ]
+        });
+        this.action({
+            title: 'Doc Holliday',
+            playType: ['shootout'],
+            cost: ability.costs.bootSelf(),
+            targets: {
+                receiver: {
+                    activePromptTitle: 'Select dude to receive bonus',
+                    cardCondition: { location: 'play area', participating: true },
+                    cardType: ['dude']
+                },
+                spellcaster: {
+                    activePromptTitle: 'Select spellcaster',
+                    cardCondition: { location: 'play area', participating: true, condition: card => card.isSpellcaster() },
+                    cardType: ['dude'],
+                    autoSelect: true                    
+                }
+            },
+            handler: context => {
+                let bulletBonus = context.targets.spellcaster.getSkillRating('huckster');
```

Comment:
Might want to put a TODO to remember to come back and make this work with all spellcaster skills.